plugins/woocommerce-admin/client/tasks/fills/steps/location.tsx

```diff
@@ -95,9 +122,14 @@ const StoreLocation = ( {
 			onSubmit={ onSubmit }
 			validate={ validate }
 		>
-			{ ( { getInputProps, handleSubmit, setValue } ) => (
+			{ ( {
+				getInputProps,
+				handleSubmit,
+				setValue,
+			}: FormContext< FormValues > ) => (
 				<Fragment>
 					<StoreAddress
+						// @ts-expect-error
```

Comment:
It seems that we need to add a description here to explain why the `@ts-expect-error` is necessary.

![Screenshot 2023-03-17 at 13 09 10](https://user-images.githubusercontent.com/4344253/225817890-7ad9bd01-2b07-4085-938b-8a71b5a0ac01.png)

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/37257/commits/2fc4213769ab1ea316cf2f53efa8d909801ce224